### ISSUE TITLE
Have a dictionary section

### ISSUE THREAD

**Issue by JMante1** [SynBioHub/synbiohub.github.io#5]
Have an additional section which just provides common terms and their definitions. Could include:
-uri
-plugin
-web of registries
-component
-fasta
-gff3
-sbol
-other file types

**Comment by PRUBHTEJ** [comment]
That sounds good, but what I've done is to provide the definitions with each term as well as provide hyperlinks for further reference. Wouldn't that suffice?

**Comment by JMante1** [comment]
I think you should do both so that if a user can just refresh what something is without finding it in the documentation where it is defined

**Comment by PRUBHTEJ** [comment]
Ohkay!
I'll try to include a section titled, "Glossary".

**Comment by PRUBHTEJ** [comment]
So, should I add a separate section or try and include this in the any other section?

**Comment by JMante1** [comment]
I would add a separate section

**Comment by PRUBHTEJ** [comment]
For this should we add a childpage under the User documentation section?

**Comment by JMante1** [comment]
No, I would add as a fully separate section as URIs are also useful for plugins etc.

**Comment by PRUBHTEJ** [comment]
Okay, once I've done that, I'll put up an update.

**Comment by PRUBHTEJ** [comment]
I've done the needful. But I think there is a possibility for further expansion of this section. Kindly have a look.

**Comment by PRUBHTEJ** [comment]
@JMante1 
Kindly have a look!
